lib/carbonite/migrations.ex

```diff
@@ -261,28 +261,32 @@ defmodule Carbonite.Migrations do
   # ------------------------------- outbox setup -----------------------------------
 
   @type outbox_name :: String.t()
-  @type outbox_option :: {:carbonite_prefix, prefix()}
+  @type outbox_option :: {:carbonite_prefix, prefix()} | {:last_transaction_id, non_neg_integer()}
 
   @doc """
   Inserts an outbox record into the database.
 
   ## Options
 
   * `carbonite_prefix` is the schema of the audit trail, defaults to `"carbonite_default"`
+  * `last_transaction_id` allows to start the outbox processing *after* the given transaction id
   """
   @doc since: "0.4.0"
   @spec create_outbox(outbox_name()) :: :ok
   @spec create_outbox(outbox_name(), [outbox_option()]) :: :ok
   def create_outbox(outbox_name, opts \\ []) do
     carbonite_prefix = Keyword.get(opts, :carbonite_prefix, default_prefix())
+    last_transaction_id = Keyword.get(opts, :last_transaction_id, 0)
```

Comment:
isn't there already a default defined in the migration? 
Also shouldn't it better be `nil` anyway in case it is the first transaction?

Reply:
there's a default on the column yeah, but it's `0` for whatever reason 🤷 probably b/c Postgres sequences start at 1 or so? Don't know what I was thinking back then.

Also, yes, there's a default already, but how would I fall back to that in the insert statement above? Would have to be an entirely different insert, right? Or is there a value like `DEFAULT` in postgres/SQL?